### PR TITLE
Update bnd files in order to remove warnings from build logs.

### PR DESCRIPTION
**Pull request by jhanders34**
- Remove extra commas at the end of a property causing Empty clause
warnings
- Remove duplicate classpath entries in buildpath causing Duplicate name
warnings.
- Remove non existent packages for Private package, Import package and Export package to
remove Unused warnings.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#build

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.ws.jaxws.ejb/bnd.bnd [hunk 1]
```diff
@@ -47,13 +47,12 @@ Service-Component: \
...
      provide:='com.ibm.wsspi.ejbcontainer.WSEJBHandlerResolver'; \
      configuration-policy:=optional; \
-     properties:="service.vendor=IBM", \
+     properties:="service.vendor=IBM"
 
 Export-Package: \
     com.ibm.ws.jaxws.ejb
 
 Import-Package: \
-   javax.xml.bind.*;version="[2.2,3)", \
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxws.ejb/bnd.bnd]: I'm pretty sure these imports were needed for Java 11 compatibility. Before you deliver these changes, be sure to do a Java 11 personal build.

Same comment applies to all other `javax.xml.bind` removals

Comment (jhanders34) [reply dev/com.ibm.ws.jaxws.ejb/bnd.bnd]: I have compared the bundle manifest files for all of open liberty image from builds before my change and after my changes.  The Import and Export package statements are identical.  Only the Private packages have differences.  The private packages have differences for packages that I removed that don't exist or have a changed order and are the same.

**Comment by jhanders34** [comment]
I compared two builds built in the build systems and even the private packages were the same except for the non existent packages that I removed.

**Pull request merged**